package.json

```diff
@@ -50,6 +50,7 @@
     "webpack-node-externals": "^1.5.4"
   },
   "dependencies": {
-    "hpq": "^1.1.1"
+    "hpq": "^1.1.1",
+    "lodash": "^4.17.4"
```

Comment:
>Could we add some technical guidelines to the docs of this project to explain some choices? React, JSX, how we use lodash, ...

We have a few notes about the first two in [Element docs](https://github.com/WordPress/gutenberg/tree/master/element). I agree it would be a good idea, even for our own benefit to ensure our choices are well reasoned.